loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
I think the Github API doesn't make a difference between closed and merged PRs, but you could look at it to try to improve it.